grid/grid.css

```diff
@@ -0,0 +1,22 @@
+.grid-50-50{
+    display: grid;
+    grid-template-columns: 50% 50%;
+    grid-template-rows: auto;
+    grid-template-areas: "left-content right-content";
+}
+
+.grid-70-30{
+    display: grid;
+    grid-template-columns: 70% 30%;
+    grid-template-rows: auto;
+    grid-template-areas: "left-content right-content";
+}
```

Comment:
this is not necessary, as you only have two elements in grid wrapper, so it will take it by default